src/definition/settings/SettingType.ts

```diff
@@ -7,4 +7,5 @@ export enum SettingType {
     SELECT = 'select',
     STRING = 'string',
     MULTI_SELECT = 'multiSelect',
+    PASSWORD = 'password',
```

Comment:
We probably need to add documentation to this to state it doesn’t encrypt the value or anything like that.

Reply:
```suggestion
	// Renders an input of type 'password' in the form. IMPORTANT - the value will NOT be encrypted
	// it will be treated as a password just on the screen
    PASSWORD = 'password',
```